### ISSUE TITLE
Argument to path.extname must be a string

### ISSUE THREAD

**Issue by jjsainzc** [Glavin001/atom-beautify#1514]
Why is too hard  ? Too many errors, I keep my sublime text

```
* Object.extname -               C:\Users\ROG\AppData\Local\atom\app-1.14.1\resources\app.asar\src\electron-shims.js:20:10
* <unknown> -                    C:\Users\ROG\.atom\packages\atom-beautify\src\beautifiers\index.coffee:230:34
* Promise._execute -             C:\Users\ROG\.atom\packages\atom-beautify\node_modules\bluebird\js\release\debuggability.js:300:9
* Promise._resolveFromExecutor - C:\Users\ROG\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:481:18
* new Promise -                  C:\Users\ROG\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:77:14
* <unknown> -                    C:\Users\ROG\.atom\packages\atom-beautify\src\beautifiers\index.coffee:226:18
```

**Comment by dewey** [comment]
Same error here

**Comment by Glavin001** [comment]
Duplicate of #1423. Also PR #1573 has been merged.

Published to v0.29.18
